modules/fedimint-meta-tests/src/bin/meta-dummy-test.rs

```diff
@@ -0,0 +1,12 @@
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    devimint::run_test(|process_mgr| async move {
+        let dev_fed = devimint::devfed::DevJitFed::new(&process_mgr)?;
+
+        dev_fed.finalize(&process_mgr).await?;
+        let client = dev_fed.internal_client().await?;
+        client.balance().await?;
+        Ok(())
+    })
+    .await
+}
```

Comment:
Can we actually test something about the meta module here? Currently it's more of a test framework test.

Reply:
Now that I know we're fine, I've converted #4772 to this approach.